### PR TITLE
Output.md updates

### PR DESCRIPTION
**Pull request by danilodileo**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metatdenovo/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/metatdenovo _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

I updated the Output.md documentation.

### CODE REVIEW

File: docs/output.md [hunk 1]
```diff
@@ -2,44 +2,80 @@
...
 
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
-
 ## Pipeline overview
 
 The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
 
-- [FastQC](#fastqc) - Raw read QC
-- [MultiQC](#multiqc) - Aggregate report describing results and QC from the whole pipeline
+- [Summary tables folder](#summary-tables-folder) - Final tables that can be imported directly in R
```
Comment (erikrikarddaniel) [inline docs/output.md:13]: I think this part of the document is summarizing the steps and not the directory structure, right? In other words, this should go somewhere else I think.

**Review by erikrikarddaniel — APPROVED** [review]
Good start. I made one comment but I think we'll have to review this looking at how other pipelines have organized this, so I'm approving.

**Comment by danilodileo** [comment]
I used Ampliseq as template

File: docs/output.md (hunk 1 above)
Comment (danilodileo) [reply docs/output.md:13]: By looking at Ampliseq they have similar structure for tables as well
https://nf-co.re/ampliseq/2.7.0/docs/output

they call it "pipeline summary report"

**Pull request merged**